README.md

```diff
@@ -1,22 +1,30 @@
 Conjur API for Java
 ===================
+Conjur API for Java SDK (OSS and Enterprise/DAP). The Java API SDK allows developers to build new Java apps that communicate with Conjur by invoking our Java API to perform operations (add, retrieve, etc) on data stored as needed.
 
-## Installation
+## Table of Contents
+- [Prequisites](#prerequisites)
+- [Setup](#setup)
+- [Configuration](#configuration)
+- [Setup Trust Between App and Conjur](#setup-trust-between-app-and-conjur)
+- [Authorization Examples](#authorization-examples)
+- [Conjur Services Operation Examples](#conjur-services-operation-examples)
+- [JAX-RS Implementations](#jax-rs-implementations)
+- [Contributing](#contributing)
+- [License](#license)
 
-### From Source
+##Prerequisites
+It is assumed that Conjur (OSS or Enterprise/DAP) and the Conjur CLI have already been installed in the environment and running in the background. If you haven't done so, follow these instructions for installation of the [OSS](https://docs.conjur.org/Latest/en/Content/OSS/Installation/Install_methods.htm) and these for installation of [Enterprise/DAP](https://docs.cyberark.com/Product-Doc/OnlineHelp/AAM-DAP/Latest/en/Content/Deployment/platforms/platforms.html).
 
-To build the library from source you'll need Maven.  You can build it like this:
+Once Conjur and the Conjur CLI are running in the background, you are ready to start setting up your Java app to work with our Conjur Java API!
```

Comment:
Do you really need the CLI to set up the Java API?

Reply:
Ideally yes because the README explains how to add the app into Conjur and they do that thru policy that is loaded in the CLI. Also, you need the CLI initialized to set up the certs b/w app and Conjur